doc/user/radial_and_axial_expansion.rst

```diff
@@ -60,27 +47,140 @@ length, :math:`L_0`,
 
-Equation :eq:`linearExpansionFactor` is the expression used by ARMI in
-:py:meth:`linearExpansionFactor
-<armi.materials.material.Material.linearExpansionFactor>`.
+Equation :eq:`linearExpansionFactor` is the expression used by ARMI in :py:meth:`linearExpansionFactor <armi.materials.material.Material.linearExpansionFactor>`.
 
 .. note::
     :py:meth:`linearExpansionPercent
     <armi.materials.material.Material.linearExpansionPercent>` returns
     :math:`\frac{L - L_0}{L_0}` in %.
+
+Given that thermal expansion (or contraction) of solid components must conserve mass throughout the system, the density of the component is adjusted as a function of temperature based on the following equation, assuming isotropic thermal expansion.
+
+.. math::
+    \rho(T_h) \frac{kg}{m^3} = \frac{\rho(T_0) \frac{kg}{m^3}}{1 + \frac{\Delta L}{L_0}} = \frac{\rho(T_0) \frac{kg}{m^3}}{(1 + \alpha_m (T_h) (T_h - T_0))^3}
+
+where, :math:`\rho(T_h)` is the component density at the given temperature :math:`T_h`, :math:`\rho(T_0)` is the component density at the reference temperature :math:`T_0`, and :math:`\alpha(T_h)` is the mean coefficient of thermal expansion at the specified temperature :math:`T_h` relative to the material's reference temperature.
+
+An update to mass densities is applied for all solid components given the assumption of isotropic thermal expansion.  Here we assume the masses of non-solid components (e.g., fluids or gases) are allowed to change within the reactor core model based on changes to solid volume changes. For instance, if solids change volume due to temperature changes, there is a change in the amount of volume left for fluid components.
+
+Implementation Discussion and Example of Radial and Axial Thermal Expansion
+===========================================================================
+
+This section provides an example thermal expansion calculation for a simple cylindrical component in the following figure:
+
+.. figure:: /.static/axial_expansion_simple.png
+
+    Illustration of Simple Cylinder Radial and Axial Thermal Expansion
+
+from a reference temperature of 20°C to 1000°C with example material properties and dimensions as shown in this table:
+
+.. list-table:: Example Component Properties for Thermal Expansion
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Property
+     - Example
+   * - Material
+     - Steel
+   * - Radius
+     - 0.25 cm
+   * - Height
+     - 5.0 cm
+   * - Reference Temperature
+     - 20°C
+   * - Density
+     - 1.0 g/cc
+   * - Mean Coefficient Thermal Expansion
+     - 2 x 10-6 1/°C
+
+This process is performed iteratively for each component as temperatures change. One important note about the thermal expansion implementation is that components are constrained within blocks, so the height of the component is determined by the height of its parent Block. What follows is a simple example illustrating the behavior of thermal expansion for a Block containing a single Component.
```

Comment:
```suggestion
Thermal expansion calculations are performed for each component in the ARMI reactor data model as component temperatures change. Since components are constrained within blocks, the height of components are determined by the height of their parent block. :numref:`hot_radius` through :nuref:`hot_density` illustrate how the radius, height, volume, density, and mass are updated for a component during thermal expansion.
```